src/autocomplete/index.ts

```diff
@@ -116,11 +118,16 @@ export interface FunctionsSuggestion {
     types?: string[];
 }
 
-export interface FiltersSuggestion {
+export interface CommonSuggestion {
     prefix?: string;
     tables: Table[];
 }
```

Comment:
Common suggestion looks too general. Why not just create a FiltersSuggestion, and make GroupBysSuggestion, OrderBysSuggestion aliases?

Or else we need a better naming